graphql/typeDefs.js

```diff
@@ -406,6 +406,7 @@ module.exports = gql`
 
   type Mutation {
     register(registerInput: RegisterInput): User!
+    deleteUser(deleteId: String!, permUser: String!): String!
```

Comment:
Actually, can you return an array of users instead? Usually all of our delete mutations work like that